generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -2,13 +2,20 @@
 <% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
 import java.time.ZonedDateTime;<% } %>
 
+import java.util.Set;
+
 import <%=packageName%>.domain.User;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 /**
  * A DTO extending the UserDTO, which is meant to be used in the user management UI.
  */
 public class ManagedUserDTO extends UserDTO {
 
+    public static final int PASSWORD_MIN_LENGTH = 5;
```

Comment:
@deepu105 and @notflorian Just a quick question... I was looking to `generators/client/templates/src/main/webapp/app/account/register/register.html` file and I've found this snippet:
```
<div class="form-group">
                    <label class="control-label" for="password" translate="global.form.newpassword">New password</label>
                    <input type="password" class="form-control" id="password" name="password" placeholder="{{'global.form.newpassword.placeholder' | translate}}"
                           ng-model="vm.registerAccount.password" ng-minlength=5 ng-maxlength=50 required>
                    <div ng-show="form.password.$dirty && form.password.$invalid">
                        <p class="help-block"
                               ng-show="form.password.$error.required" translate="global.messages.validate.newpassword.required">
                            Your password is required.
                        </p>
                        <p class="help-block"
                               ng-show="form.password.$error.minlength" translate="global.messages.validate.newpassword.minlength">
                            Your password is required to be at least 5 characters.
                        </p>
                        <p class="help-block"
                               ng-show="form.password.$error.maxlength" translate="global.messages.validate.newpassword.maxlength">
                            Your password cannot be longer than 50 characters.
                        </p>
                    </div>
                    <password-strength-bar password-to-check="vm.registerAccount.password"></password-strength-bar>
                </div>
```
In this file, the min length for password field is 5. Is that correct? 
If we leave the min password length to 4, in order to avoid backward compatibility, we will have different validation rules to password field for back-end and front-end Someone could clarify this question?